src/Uno.Playground.WASM/Uno.Playground.WASM.csproj

```diff
@@ -103,6 +103,7 @@
 		<PackageReference Include="Microsoft.Extensions.Logging.Console" Version="6.0.0" />
 		<PackageReference Include="Microsoft.Extensions.Logging.Filter" Version="1.1.2" />
 		<PackageReference Include="Microsoft.Windows.Compatibility" Version="6.0.0" />
+		<PackageReference Include="CommunityToolkit.Graph.Uwp.Controls" Version="7.1.1" />
```

Comment:
This is not going to work unfortunately. I remembered that we don't have that package available for UWP, only for WinUI. This means we can't do this yet for the playground... you can close this PR.